twilio/task_router/__init__.py

```diff
@@ -209,13 +227,15 @@ def __init__(self, account_sid, auth_token, workspace_sid, worker_sid):
                                                          workspace_sid,
                                                          worker_sid)
 
-        self.reservations_url = self.base_url + "/Tasks/**"
         self.activity_url = self.base_url + "/Activities"
+        self.tasks_url = self.base_url + "/Tasks/**"
```

Comment:
I do believe that this would be very rare behavior for any user. However, if there is any user doing this - it would be a breaking change, which would require a major version bump.